Assets/MRTK/Core/Providers/InputSimulation/SimulatedArticulatedHand.cs

```diff
@@ -122,6 +142,9 @@ protected override void UpdateInteractions(SimulatedHandData handData)
                             CoreServices.InputSystem?.RaisePoseInputChanged(InputSource, ControllerHandedness, Interactions[i].MixedRealityInputAction, currentIndexPose);
                         }
                         break;
+                    case DeviceInputType.ThumbStick:
+                        handDefinition.UpdateCurrentTeleportPose(Interactions[i]);
```

Comment:
Does this mean that it's only possible to simulate an articulated hand using a thumbstick? What if I only have a keyboard and mouse, how would I simulate the teleport gesture?

Reply:
addressed mostly up here: https://github.com/microsoft/MixedRealityToolkit-Unity/pull/8977#discussion_r532993003

The current plan to get this working with InputSimulation is to have the hand strike a Teleport Pose, after which the articulated hand definition will handle raising the appropriate events to start the teleportation process.